project/SparkBuild.scala

```diff
@@ -1580,6 +1580,19 @@ object TestSettings {
       "SPARK_TESTING" -> "1",
       "JAVA_HOME" -> sys.env.get("JAVA_HOME").getOrElse(sys.props("java.home")),
       "SPARK_BEELINE_OPTS" -> "-DmyKey=yourValue"),
+
+    // Copy system properties to forked JVMs so that tests know proxy settings
```

Comment:
No biggie but I would place the JIRA number here, e.g.)

```suggestion
    // Copy system properties to forked JVMs so that tests know proxy settings, see SPARK-44110.
```